UnitTestProject/PokemonTest.cs

```diff
@@ -218,7 +224,11 @@ public void Pokemon_CombinedEV_Fail_GreaterThan_EV_MaxLimit()
 			//All EV points when added together cannot be greater than a sum of MaxEVLimit
 			Pokemon pokemon = new Pokemon();
 			int ev = pokemon.EV[0] + pokemon.EV[1] + pokemon.EV[2] + pokemon.EV[3] + pokemon.EV[4] + pokemon.EV[5];
-			Assert.AreEqual(Pokemon.EVLIMIT, ev);
+            if (Pokemon.EVLIMIT < ev)
+            {
+                Assert.Fail("EV Limit is less than the total EV level");
+            }
+            Assert.IsTrue(Pokemon.EVLIMIT >= ev);
```

Comment:
lines 224: `//All EV points when added together cannot be greater than a sum of MaxEVLimit`
`public void Pokemon_CombinedEV_Fail_GreaterThan_EV_MaxLimit()`
I think you hit the goal for what you were testing xD But when a new pokemon is created, all values are zero'd out and empty. The test is failing, not because the value is under the cap. It's failing, because the stats are empty, and they should be filled until they reach their limit. Their limit is, when they equal threshold.
The test is, values can _not_ be greater than max limit. So automatically passing "equal to or greater than" doesnt do anything for the test. Add some arbitrary numbers, some really wild numbers (as if you were able to hack or modify the game) and see if it breaks (fail) on you when you run the test again.